### PR TITLE
Combines Xenoarch Gas Effects into a single effect.

### PR DESCRIPTION
**Pull request by gurfan**
Also fixes them producing so little gas you don't even notice what the fuck they're doing.

Personally I would rather delete the gas effect and the gas trigger from existence, but r*moval is a dangerous thing. 
This PR merges the gas generation effects into a single type. Instead of being five separate effects, there is a single gas effect type that will pick a random gas to emit when it is created. In-game this just means gas effects aren't 5x more likely to occur than any other effect. **A single will not generate more than one type of gas, as usual.** (but let me know if you'd rather see multiple gases generated at once)

While testing I noticed that the artifacts were producing so little gas it wouldn't even show up on analyzers. I tweaked the numbers so its clearly noticeable now. Their gas production is still largely outclassed by gas miners anyway.

[tweak]

:cl:
 * tweak: Merged Xenoarcheology gas effect into a single type.
 * tweak: Increased Xenoarcheology gas effect generation amount so it's actually noticeable.

### CODE REVIEW

File: code/modules/research/xenoarchaeology/artifact/triggers/unknown_trigger_gas.dm [hunk 1]
```diff
@@ -3,21 +3,21 @@
 /datum/artifact_trigger/gas
 	triggertype = TRIGGER_GAS
 	scanned_trigger = SCAN_ATMOS
-	var/trigger_gas = 0
+	var/trigger_gas = null
 
 /datum/artifact_trigger/gas/New()
 	..()
-	trigger_gas = pick("PLASMA", "CARBON_DIOXIDE", "NITROGEN", "OXYGEN") //TODO: See if these can be replaced with the defines without fucking up
+	trigger_gas = pick(GAS_NITROGEN, GAS_OXYGEN, GAS_CARBON, GAS_PLASMA) //TODO: delete this comment
```
Comment (jknpj) [inline code/modules/research/xenoarchaeology/artifact/triggers/unknown_trigger_gas.dm]: delet this comment

**Comment by jknpj** [comment]
maps/synergy.dmm:4922:error: undefined type: /obj/item/weapon/rcd_ammo

**Comment by gurfan** [comment]
uhhhhhhhhhhhhh

**Comment by gurfan** [comment]
> maps/synergy.dmm:4922:error: undefined type: /obj/item/weapon/rcd_ammo

#27497 @jknpj

**Comment by DrSnips** [comment]
👍 For removing gas based TRIGGERS if you want to get a feel for support.

**Comment by jknpj** [comment]
Give them a proper gas chamber instead. Plasma generation on O2 is a god-given right

**Pull request merged**